src/Common/tests/System/PlatformDetection.cs

```diff
@@ -118,6 +119,11 @@ private static IdVersionPair ParseOsReleaseFile()
                 ret.VersionId = versionId.Substring(1, versionId.Length - 2);
             }
 
+            if (ret.Id[0] == '"' && ret.Id[ret.Id.Length - 1] == '"')
```

Comment:
The id here is always expected to be > 2, since it's the full distro name. Version Id, can be < 2, as it only contains the version number. But will add the check.